browser-test/src/support/admin_predicates.ts

```diff
@@ -43,6 +43,11 @@ export class AdminPredicates {
     )
   }
 
+  async updateEligibilityMessage(eligibilityMsg: string) {
+    await this.page.fill('input[name=eligibilityMessage]', eligibilityMsg)
```

Comment:
**suggestion (non-blocking):** We've been trying to use the `.getByRole` locator so that we're locating elements in the same way a screen reader would. So, I think you could do something like:

`await this.page.getByRole('input', {name: `eligibilityMessage`})`
`await this. page.getByRole('button', {name: 'Save eligibility message'})`

Reply:
makes sense! changes applied.